packages/epics/src/schedule/components/space-calendar.css

```
@@ -0,0 +1,181 @@
+/* Hypha space calendar — FullCalendar theme integration */
+
+.hypha-space-calendar .fc {
+  --fc-border-color: hsl(var(--border) / 0.55);
+  --fc-page-bg-color: transparent;
+  --fc-neutral-bg-color: hsl(var(--muted) / 0.35);
+  --fc-neutral-text-color: hsl(var(--muted-foreground));
+  --fc-today-bg-color: hsl(var(--primary) / 0.08);
+  --fc-event-border-color: transparent;
+  --fc-now-indicator-color: hsl(var(--primary));
+  font-family: inherit;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)